src/js/hca/config/migrations.js

```diff
@@ -50,5 +50,17 @@ export default [
     }
 
     return { formData: newData, metadata };
+  },
+  // 2 -> 3, we're updating the url for the dependent info page since it's for dependents
+  // and not just children anymore
+  ({ formData, metadata }) => {
+    const url = metadata.returnUrl || metadata.return_url;
```

Comment:
Would `metadata` ever contain the camelcase version of these two, should only the snake case version be used here? The camelcase version is available on formData I believe, but I'm not sure it is available on metadata.

Reply:
The vast majority of the time it should be camel case. The front end only ever reads and writes the return url in camel case. I believe the backend will send it in snake case when prefilling data, which the front end ignores in general, but I think we have to account for in migrations.